__tests__/components/App.test.js

```diff
@@ -4,14 +4,18 @@ import 'jsdom-global/register'
 import React from 'react'
 import { mount, shallow } from "enzyme"
 import { MemoryRouter } from "react-router"
-import RootContainer from '../../src/components/RootContainer'
+import { Provider } from 'react-redux'
 import App from '../../src/components/App'
-import Config from '../../src/Config'
+import LoginPanel from '../../src/components/LoginPanel'
 import HomePage from '../../src/components/HomePage'
 import Editor from '../../src/components/editor/Editor'
 import Browse from '../../src/components/editor/Browse'
+import CanvasMenu from '../../src/components/CanvasMenu'
 import Footer from '../../src/components/Footer'
-import ImportResourceTemplate from "../../src/components/editor/ImportResourceTemplate";
+import ImportResourceTemplate from '../../src/components/editor/ImportResourceTemplate';
+
+jest.mock('../../src/components/editor/Editor')
+jest.mock('../../src/components/editor/ImportResourceTemplate')
```

Comment:
similarly, `ImportResourceTemplate` ultimately makes a call out to the server (via the `SinopiaResourceTemplates` component) to see if the server has new info for display.  these `App` unit tests are just about whether `ImportResourceTemplate` displays iff the user is logged in *and* the user goes to `/templates`.  we can write full on headless browser integration tests with a backend if we want to make sure that things work from UI to server and back.